build_tools/android/upload_benchmarks_to_dashboard.py

```diff
@@ -0,0 +1,291 @@
+#!/usr/bin/env python3
+# Lint as: python3
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Upload benchmark results to IREE Benchmark Dashboards.
+
+This script is meant to be used by Buildkite for automation.
```

Comment:
Do we have any ability to run this locally (or instructions on how to update it/test)? Just thinking of 11pm me working through an issue :)

Reply:
I think you should be fine 11pm. Normally not expecting this to be touched given we can control both side (Buildkite and Dana) and APIs are expected to be stable. But yeah. I added a `--dry-run` option to help on locally run it to see all the API call payloads. (Debugging locally need a local Dana server. I should write a doc about how to play with that later.)